Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs

```diff
@@ -17,6 +18,34 @@ public CandidateRepository(IOptions<MongoSettings> options)
         {
         }
 
+        public Task<ProjectResult> GetCandidateWithProject(Guid id, Guid projectId)
+        {
+            return GetCollection().Aggregate().Match(c => c.Id == id)
+               .Unwind<Candidate, ProjectResult>(c => c.ProjectResults).Match(p => p.ProjectId == projectId)
+                .Project(p =>
+                   new ProjectResult
+                   {
+                       Status = p.Status,
+                       Feedbacks = p.Feedbacks,
+                       ProjectId = p.ProjectId,
+                       Reason = p.Reason,
+                   }).FirstOrDefaultAsync();
+        }
+
+        public Task<ProjectResult> GetCandidateWithProjectFeedbackAsync(Guid id, Guid projectId, Guid feedbackUserId, FeedbackType feedbackType)
+        {
+            return GetCollection().Aggregate().Match(c => c.Id == id)
```

Comment:
.Match from new line, and fix in both methods other places with dot too

Reply:
ready